chia/rpc/wallet_rpc_api.py

```diff
@@ -1404,6 +1405,20 @@ async def nft_get_by_did(self, request) -> Dict:
                 return {"wallet_id": wallet.wallet_id, "success": True}
         return {"error": f"Cannot find a NFT wallet DID = {did_id}", "success": False}
 
+    async def nft_get_wallet_did(self, request) -> Dict:
+        wallet_id = uint32(request["wallet_id"])
+        assert self.service.wallet_state_manager is not None
+        nft_wallet: NFTWallet = self.service.wallet_state_manager.wallets[wallet_id]
+        if nft_wallet is not None:
+            if type(nft_wallet) != NFTWallet:
```

Comment:
type() compared to the uint8 of this enum member?

Reply:
NFTWallet doesn't expose a WalletType property that I see. There's a classmethod for returning the WalletType, but it doesn't apply to instances.

Reply:
Nevermind, I was mistaken about how @classmethod works. Fixed now.

Reply:
you could use `type()` here: https://github.com/Chia-Network/chia-blockchain/blob/release/1.4.0/chia/wallet/nft_wallet/nft_wallet.py#L128

`nft_wallet.type() == WalletType.NFT`